### PR TITLE
Code formatting

### PR DESCRIPTION
**Pull request by EDsCODE**
- commit hook that will run prettier on staged changes
- ran prettier on all file (js,ts,html,css,jsx)

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -65,11 +65,13 @@
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-to-json": "^3.3.4",
+    "husky": "^4.2.3",
```
Comment (tristanhcole) [inline app/package.json]: highlighting packages installed

Comment (tristanhcole) [reply app/package.json]: - husky
- pretty-quick

**Comment by tristanhcole** [comment]
@enjeyw how does this compare to any existing git hooks?

**Comment by enjeyw** [comment]
> @enjeyw how does this compare to any existing git hooks?

We don't actually use any git hooks in the current build, so this _should_ be fine

**Comment by enjeyw** [comment]
> > @enjeyw how does this compare to any existing git hooks?
> 
> We don't actually use any git hooks in the current build, so this _should_ be fine

Mine didn't run at first, ran 
` git rev-parse --git-path hooks` and got `.githooks`
ran
`git config core.hooksPath .git/hooks/` and everything behaved

**Comment by tristanhcole** [comment]
[Setup linter to standardize code styling!](https://trello.com/c/rFzwJZHg/257-setup-linter-to-standardize-code-styling)

**Pull request merged**